cxx/structureTranslationModule/translator/FromConceptTranslator.cpp

```diff
@@ -0,0 +1,39 @@
+#include "FromConceptTranslator.hpp"
+
+namespace structureTranslationModule
+{
+FromConceptTranslator::FromConceptTranslator(ScMemoryContext * context)
+  : StructureTranslator(context)
+{
+}
+
+std::vector<std::string> FromConceptTranslator::translate(ScAddr const & structAddr) const
```

Comment:
Why do you need std::vector<std::string>? Could you replace it with just string or stringstream?

Reply:
Change everywhere in module